README.md

```diff
@@ -24,6 +24,7 @@ https://groups.google.com/a/opencontainers.org/forum/#!forum/tob (tob@opencontai
 * [Image Format Spec](https://github.com/opencontainers/tob/tree/master/proposals/image-format)
 * [SELinux](https://github.com/opencontainers/tob/blob/master/proposals/selinux.md)
 * [Tools](https://github.com/opencontainers/tob/blob/master/proposals/tools.md)
+* [libocispec](https://github.com/opencontainers/tob/blob/master/proposals/libocispec.md)
```

Comment:
Previous entries in this list seem to be alphabetized, although I don't know if TOB maintainers care about that.